src/wp-content/themes/twentynineteen/inc/color-patterns.php

```diff
@@ -48,24 +48,24 @@ function twentynineteen_custom_colors_css() {
 	$lightness = absint( $lightness ) . '%';
 
 	/**
-	 * Filters Twenty Nineteen default hover lightness level.
+	 * Filters Twenty Nineteen default reduced lightness level.
 	 *
 	 * @since Twenty Nineteen 1.0
 	 *
-	 * @param int $lightness_hover Hover color lightness level.
+	 * @param int $lightness_reduced Reduced color lightness level.
 	 */
-	$lightness_hover = apply_filters( 'twentynineteen_custom_colors_lightness_hover', 23 );
-	$lightness_hover = absint( $lightness_hover ) . '%';
+	$lightness_reduced = apply_filters( 'twentynineteen_custom_colors_lightness_reduced', 23 );
+	$lightness_reduced = absint( $lightness_reduced ) . '%';
```

Comment:
Renamed this to be more generic as it's used for hover, active and focus, and now secondary border color too.